package.json

```diff
@@ -42,7 +43,6 @@
     "url": "https://github.com/angular/angular-cli.git"
   },
   "engines": {
-    "node": ">=10.3.0 <11.0.0",
```

Comment:
why removed? bad merge?

Reply:
Not sure how it got there. Glad you caught it!